spa/src/components/molecules/InfoBox.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div class="info-box"  v-if="infoBoxContent">
+    <span class="closebtn" @click="ignoreMessage()"><BIconXCircle scale='1'></BIconXCircle></span>
+    <component v-if="infoBoxContent" :is="infoBoxContent"></component>
+  </div>
+</template>
+
+<script>
+import { BIconXCircle } from 'bootstrap-vue';
+import { infoBoxes } from '../../../config/infoBox.json';
+
+export default {
+  name: 'InfoBox',
+  components: {
+    BIconXCircle,
+  },
+  data() {
+    return {
+      infoBoxContent: null,
+      infoBoxId: null,
+    };
+  },
+  mounted() {
+    this.loadInfoBox();
+  },
+  methods: {
+    loadInfoBox() {
+      const userIgnored = this.checkIfIgnored();
```

Comment:
this variable is used only in if, it can be inlined

Reply:
Done